prisma/schema/teamStack.prisma

```diff
@@ -2,13 +2,20 @@
 
 model TechStackCategory {
   id          Int    @id @default(autoincrement())
-  name        String @unique
+  name               String             @db.Citext
   description String
+  voyageTeam         VoyageTeam?         @relation("Team", fields: [voyageTeamId], references: [id], onUpdate: Cascade, onDelete: Cascade)
+  voyageTeamId       Int?
+
+  voyageTeamNew         VoyageTeam?         @relation("NewTeam", fields: [voyageTeamIdNew], references: [id], onUpdate: Cascade, onDelete: Cascade)
```

Comment:
is voyageTeamNew required? I see we have voyageTeam and also voyageTeamNew, voyageTeamNew seems to be unused (all null) in the database

Reply:
I added voyageTeamNew because of our conversation about migrating the nullable column to a new one and then making it non-nullable - yes it is unused right now

Reply:
can it be removed now? or do we need to do it in another PR? would an additional migration work? Is this a temporary column? I think I'm sligtly confused now

Reply:
Last week you were describing a 2 step migration, starting with making a new column to copy values to.  I thought you wanted to do that in this PR.

Reply:
From [this](https://github.com/chingu-x/chingu-dashboard-be/pull/208/#discussion_r1805583201) comment

Reply:
If you can sure, sorry I haven't tried or tested anything, we might be able to do multiple migrations here like you have already did (I see 3 migrations here) if not, then we can do it in another PR, we don't want duplicated columns.

I don't think we can do it in 1 PR but I could be wrong, reason being you'll need to migrate it to add `voyageTeamId`, but we cannot make it non-nullable the first migration because it's a new column and old rows has no value, so making it nullable is the only option for this migration to success (or set a default value might work), then we have to patch null values to non-null on prod server then migrate again to update column definition to make `voyageTeamId` non-nullable

I see both voyageTeamId and newVoyageTeamId are still nullable, but you seem to be able to set a composite unique, so I think we don't need a temp column now, and we can migrate, patch the existing data, then make it non optional? 

Intially, I thought you won't be able to add `@@unique` due to voyageTeamId being optional, in this case it would be necessary to create a non-nullable newVoyageTeamId, use that instead and rename it later on.

But now I think this method would be more straightforward 
![image](https://github.com/user-attachments/assets/9226b705-172e-47e9-bf03-620f44fd0a41)

I hope this explains the situation better

Reply:
Yeah sounds good - I'll remove the new column

Reply:
Ok - I've removed the new column from the categories schema , cleaned up the migrations folder , updated changelog.  Should be good to go